pkgs/applications/misc/curaengine/default.nix

```
@@ -16,6 +16,9 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [ "-DCURA_ENGINE_VERSION=${version}" ];
 
+  # TODO already fixed in master, remove in next release
+  patches = [ ./missing-import-cstdint-when-gcc-13.diff ];
+
```

```suggestion
  # TODO already fixed in master, remove in next release
  patches = [
    (fetchpatch {
      url = "https://github.com/Ultimaker/CuraEngine/commit/de60e86a6ea11cb7d121471b5dd192e5deac0f3d.patch";
      hash = "";
      includes = [ "src/utils/math.h" ];
    })
  ];

```

using [fetchpatch](https://nixos.org/manual/nixpkgs/stable/#fetchpatch) is preferred if there is an upstream patch, which there is.

---

Oh you can add `includes`, didn't know. I didn't want to use the entire commit as patch because it also does other things.
Done, thanks!